snuba/cli/subscriptions.py

```diff
@@ -138,14 +142,12 @@ def subscriptions(
         SynchronizedConsumer(
             KafkaConsumer(
                 build_kafka_consumer_configuration(
-                    bootstrap_servers,
-                    consumer_group,
-                    auto_offset_reset=auto_offset_reset,
+                    broker_config, consumer_group, auto_offset_reset=auto_offset_reset,
```

Comment:
Did you skip the commit hooks? there should be a commit hook that ensure black style is applied. If not could you please ensure you ran `make setup-git` as explained here: https://github.com/getsentry/snuba#install--run-only-required-if-you-are-developing-against-snuba

Reply:
I have the commit hooks installed, did not skip them. Let me call the hook manually to refactor it.

Reply:
Called ```black snuba/cli/subscriptions.py``` and it only removed the trailing comma.

Reply:
You are right, you made the statement just short enough that it fits on a line. Please ignore the comment.